src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -135,7 +135,7 @@ object SamResourceTypes {
     resourceId: ResourceId,
     policies: Map[AccessPolicyName, AccessPolicyMembership],
     authDomain: Set[WorkbenchGroupName],
-    returnResource: Boolean = false)
+    returnResource: Option[Boolean] = Some(false))
```

Comment:
I don't think this needs to be an `Option`, as long as you're specifying the default via `returnResource: Boolean = false`